.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "10"
   - "12"
 install:
 - npm install
```

Comment:
If we are fixing this file we should fix the line here to use ci rather than install